packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costMangement.js

```diff
@@ -0,0 +1,222 @@
+import { TextContent, Text, TextVariants, TextListItem, TextList, Button, ClipboardCopy, Popover, ButtonVariant } from '@patternfly/react-core';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const PrerequisiteDescription = () => (
+    <TextContent key='description'>
+        <Text component={TextVariants.p}>
+            Before you begin, install the following prerequisites if not present:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPText = () => (
+    <TextContent key='prerequisite-ocp'>
+        <Text component={TextVariants.p}>
+            On your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteSystemText = () => (
+    <TextContent key='prerequisite-system'>
+        <Text component={TextVariants.p}>
+            On a system with network access to your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPList = () => (
+    <TextContent>
+        <TextList component='ul' style={{ marginLeft: '16px', marginTop: '-16px' }}>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://docs.openshift.com/container-platform/3.11/getting_started/install_openshift.html"
+                >
+                    OpenShift Container Platform 3.11 or newer
+                </Text>
+            </TextListItem>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/configuring_clusters/installing-operator-framework"
+                >
+                        Operator Metering
+                </Text>
+            </TextListItem>
+        </TextList>
+    </TextContent>
+);
+
+export const PrerequisiteSystemList = () => (
+    <TextContent>
+        <TextList component='ul' style={{ marginLeft: '16px', marginTop: '-16px' }}>
+            <TextListItem component='li'>
+                <Text component={TextVariants.a} target="_blank" href="https://access.redhat.com/products/red-hat-insights#getstarted" >Red Hat Insights Client</Text>
+            </TextListItem>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#basics-what-will-be-installed"
+                >
+                    Ansible
+                </Text>
+                {' '}and the{' '}
```

Comment:
Can we someway do it without blocks with empty strings? At least we could use html constants for spaces